pkg/orchestration/wiring/kubeingress/plugin.go

```diff
@@ -74,7 +75,13 @@ func WireUp(resource *orch_v1.StateResource, context *wiringplugin.WiringContext
 	}
 	smithResources = append(smithResources, ingressResource)
 
+	contract := wiringplugin.ResourceContract{
```

Comment:
Make it part of the literal below. Why not?

Reply:
done
it is more readable this way, but given that we use this awful nested structures initializations everywhere already I'll inline this